cmd/fscrypt/commands.go

```diff
@@ -191,12 +191,14 @@ func encryptPath(path string) (err error) {
 
 	var policy *actions.Policy
 	var recoveryPassphrase *crypto.Key
+	var recoveryProtector *actions.Protector
```

Comment:
Does the protector need to be at this outer scope, or can it be in the inner scope where we create it.

Reply:
Moved to inner scope.